Automattic/newspack-plugin
# feat(ras-acc): dynamically set otp email text color

### All Submissions:






### Changes proposed in this Pull Request:



**PR 2 of 4**

This PR adds onto the work in https://github.com/Automattic/newspack-plugin/pull/3032 by adjusting email text, button, and social icon color for the default email template to either white or black depending on background color:

![Screenshot 2024-04-04 at 17 35 31](https://github.com/Automattic/newspack-plugin/assets/17905991/d5a25d21-e325-4d12-8206-4a91c54a940c)

Note: The changes only apply to the one time password email for the time being. Once that email is fully updated, I will get a PR up to update the remaining templates.

### How to test the changes in this Pull Request:

1. Go to the WP Customizer and set a dark custom primary and secondary color
2. With these changes checked out, go to Newspack > Engagment > Show Advanced Settings and select the one-time password email under Transactional Email Content section
3. Trash the post (so that we can force our updated template to be used on next otp sign in attempt)
4. As a logged out user, select the sign in link and attempt to login via email to trigger an otp email
5. Check the otp email and confirm all of the text is properly contrasting
6. Repeat the above steps with light custom colors in customizer

### Other information:
